.github/workflows/python-package.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        pip install --upgrade pip pytest
```

Comment:
We removed upgrading pytest. Do we need to add upgrading tox since that's what we're using now?

Reply:
Even if we still used pytest instead of tox, this wouldn't be needed because pytest is installed by
```
pip install -e .[test]
```